components/question/NetPromotingScore/index.jsx

```diff
@@ -0,0 +1,66 @@
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import { useEffect, useState } from 'react';
+
+import { buildSurveyQuestionsWithAnswers } from '../Details/buildSurveyQuestionsWithAnswers';
+
+export const netPromotingScoreTestIds = {
+  base: 'rating',
+  score: 'rating__score',
+  unlikelyText: 'rating__unlikely-text',
+  likelyText: 'rating__likely-text',
+};
+
+const NetPromotingScore = ({ id, answers, setSurveyQuestionsWithAnswers }) => {
+  const [currentScore, setCurrentScore] = useState(-1);
+  const totalScore = [...Array(answers.length)];
```

Comment:
Not so sure about this but it's a bit weird that there's a question with 11 answers and we need to display 1 to 11 score, so maybe it means 0 to 10 score 🤔  (0 means the user doesn't need to select the score)
![Screen Shot 2565-05-19 at 10 49 57](https://user-images.githubusercontent.com/1772999/169200467-b5b665b1-ddb7-4237-9034-556a5219dff8.png)
![Screen Shot 2565-05-19 at 10 45 00](https://user-images.githubusercontent.com/1772999/169200473-a46bac31-8036-4389-ad08-c5c7f875d13d.png)

Reply:
done in done in 5ae532f386553121a8c8a14be8a2ae4923c0b9b3!!